llama_index/indices/vector_store/retrievers/auto_retriever/prompts.py

```diff
@@ -102,13 +101,10 @@
 DEFAULT_VECTOR_STORE_QUERY_PROMPT_TMPL = PREFIX + EXAMPLES + SUFFIX
 
 
-class VectorStoreQueryPrompt(Prompt):
-    """Vector store query prompt."""
+# deprecated, kept for backwards compatibility
```

Comment:
does it matter if we set class to `Prompt` for BW compatibility but prompt type for all these prompts will be "PromptType.CUSTOM"?

Reply:
wonder if we should just define light subclasses over Prompt that set prompt type

Reply:
Since we only use prompt type for testing purpose, seems like this doesn't matter too much?

Reply:
hmm yeah fair